odl/discr/discr_mappings.py

```diff
@@ -35,17 +38,22 @@
 from odl.space.base_ntuples import NtuplesBase, FnBase
 from odl.space.fspace import FunctionSet, FunctionSpace
 from odl.set.domain import IntervalProd
-from odl.util.vectorization import is_valid_input_meshgrid
+from odl.util.vectorization import (is_valid_input_meshgrid,
+                                    out_shape_from_meshgrid)
 
 
 __all__ = ('FunctionSetMapping',
-           'GridCollocation', 'NearestInterpolation', 'LinearInterpolation')
+           'GridCollocation', 'NearestInterpolation', 'LinearInterpolation',
+           'PerAxisInterpolation')
+
+SUPPORTED_INTERP_SCHEMES = ['nearest', 'linear']
 
 
 class FunctionSetMapping(Operator):
 
     """Abstract base class for function set discretization mappings."""
 
+    # pylint: disable=abstract-method
```

Comment:
I know and agree, but I chose this over not disabling (because the warnings are nonsense) and disabling globally since warnings can actually help to find bugs. Maybe not so much warnings like `abstract-method` and `private-access`.